### PR TITLE
Compress stringify package

### PR DESCRIPTION
**Pull request by thewtex**

### CODE REVIEW

**Comment by thewtex** [comment]
This branch is too large to reasonably request a review, but pinging to bring awareness:

- There is now runtime configuration of WebAssembly pipeline assets (defaulting to jsDelivr), and this is used here on a per-package scope.
- `bindgen` generates a package configuration, documentation, typescript, browser and node bundle configuration, and configuration for a demo application built with vite.

There is more to do, and I am sure there are many improvements we can make, but leaving that for follow-up PRs. Suggestions and improvements are welcome!

**Review by PaulHax — APPROVED** [review]
Builds and runs ResampleLabelImage in itk-vtk-viewer

File: packages/compress-stringify/CMakeLists.txt [hunk 1]
```diff
@@ -3,10 +3,7 @@ project(compress-stringify)
 
 set(CMAKE_CXX_STANDARD 17)
 
-include(FetchContent)
-if (NOT TARGET libzstd_static)
-  include(${CMAKE_CURRENT_SOURCE_DIR}/BuildZstd.cmake)
-endif()
+include(${CMAKE_CURRENT_SOURCE_DIR}/BuildZstd.cmake)
```
[the remark this answers is not in the document]
Comment (thewtex) [reply packages/compress-stringify/CMakeLists.txt:6]: Good catch :eagle: :eye: !

Yes, in general we should avoid this duplication.

We have some refactoring to do before consolidation, tracked in #716

**Comment by thewtex** [comment]
> Builds and runs ResampleLabelImage in itk-vtk-viewer

Wonderful! :tada:

**Pull request merged**